data/human/free worlds side plots.txt

```diff
@@ -808,3 +808,74 @@ mission "FW Stack Core 1C"
 		conversation
 			`You receive a message from Freya: "Hello again, Captain! Thanks for your help escorting the materials to Solace. Ever since we developed the Dreadnoughts people have been talking about how badly we need a generator that can meet their energy needs, and that's what Delta V Corporation has been working on. So, take a look next time you're on Solace or one of our other major manufacturing centers. Sorry for the mystery surrounding the cargo, but we didn't exactly want to advertise that a huge shipment of fissionable material was headed for Free Worlds space."`
 				decline
+
+
+
+mission "FW Refugees to Humanika"
+	name "Transport Refugee Family"
+	description "Transport a family of <bunks> refugees and <cargo> to <destination>. Payment is <payment>."
+	minor
+	source
+		attributes "dirt belt" rim south
+		not government Quarg
+		not near Tarazed 1
+	destination Humanika
+	to offer
+		has "event: war begins"
+		not "chosen sides"
+		random < 15
+	to fail
+		has "event: joined the free worlds"
+	passengers 3
+	cargo "household goods" 2
+	on offer
+		conversation
+			`A young family of three is walking around the spaceport talking to merchant captains. From the looks of it, they're not having much luck.`
+			choice
+				`	(Approach them.)`
+				`	(Ignore them.)`
+					defer
+			`	The parents introduce themselves as Penny Little and Rosa Perkins. Their young child is named Eduardo. "Any chance you're willing to take us and <tons> of our belongings to the Quarg planet <destination>? We're concerned that the war between the Republic and the Free Worlds is going to get dangerous, and we want to raise Eduardo somewhere safe. We can pay <payment>."`
+			label "main choice"
+			choice
+				`	"Why were all those other captains refusing to take you?"`
+					to display
+						not "FW Refugees to Humanika: other captains"
+					goto "other captains"
+				`	"What do you mean, 'Quarg planet'?"`
```

Comment:
```suggestion
				`	"What do you mean, 'Quarg planet?'"`
```

Reply:
`'Quarg planet'` is not a question, so I think the question mark belongs outside of that quote. https://www.thesaurus.com/e/grammar/does-punctuation-go-inside-or-outside-quotation-marks/
> If a sentence is a question or an exclamation, but the quote is not, we typically place the punctuation mark outside of the quotation marks.